src/electionguard_cli/print_utils.py

```diff
@@ -1,19 +1,26 @@
 import click
+from typing import Any
 
-def print_header(text: str, header_color="green"):
+def print_header_utils(text: str, header_color="green"):
     click.echo("")
     click.secho(f"{'-'*40}", fg=header_color)
     click.echo(text, fg=header_color)
     click.echo(f"{'-'*40}", header_color)
 
-def print_message(text: str, underlined=False):
+def print_message_utils(text: str, underlined=False):
     click.secho(f"{text}", fg="white", underline=underlined)
 
-def print_warning(text: str, warning_color="bright_red") -> None:
+def print_warning_utils(text: str, warning_color="bright_red") -> None:
     click.secho(f"WARNING: {text}", fg=warning_color, bold=True)
 
-def print_error(text: str, error_color="bright_red"):
+def print_error_utils(text: str, error_color="bright_red"):
```

Comment:
Whats the reason for adding _utils at the end of each function?

Reply:
inside of cli_step_base print_header was the name of the method that was already existing and just to distinguish which print_header belonged where. If I didnt add _utils it would have confused me as to which method I was using

Reply:
I understand, but those methods are in different files. to disambiguate, try using the "import as" functionality in python so that print_header in the base calls print_utils.print_header in the new print module.